.github/actions/commit_and_push/commit_and_push.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
```

Comment:
It would be really nice to make this file more declarative, it's kind of crazy

Reply:
done